### ISSUE TITLE
Async customSanitizer

### ISSUE THREAD

**Issue by ajmariduena** [express-validator/express-validator#784]
Hi guys,

Im trying to use a promise on a `customSanitizer` with the following code:

```js
.customSanitizer(async (value) => {
    const User = await User.findOne({
        where: {
            id: value,
        },
    });

    return User;
})
```

But promise is not being resolved:
`{ user: Promise { <pending> } }`

Is there a way to do this?.

**Comment by gustavohenke** [comment]
That's expected. There's no support for async sanitizers at the moment.
It shouldn't be hard to implement, though.

**Comment by ilivss** [comment]
Damn I wasted like 4 hours searching for solution untill I found this post. Is there a workaround in the meantime?

**Comment by NathanPB** [comment]
Is anyone already working on that?

**Comment by fedeci** [comment]
Fixed in #919
